### DISCUSSION TITLE
set StreamDeck icon based on entity state

### DISCUSSION THREAD

**Discussion by Elitehoernchen** [cgiesche/streamdeck-homeassistant#216] [Help and Support]
I am using a button to toggle my Smart Thermostat between "off" and "heat"

as the entity replies with "heat" instead of "on" it doenst display the on state correctly. Where in the scripts I could adjust it, so it recognises "On" or "heat" as active state?

**Comment by cgiesche** [comment]
I assume you are using custom icons for on and off state?

Currently states like "on", "open", "active" etc. are mapped to "on" not in a configuration but are hardcoded. I will add "heat" as an active state with the next release.

Tracking Issue: #224
